coco/master.py

```diff
@@ -124,7 +124,7 @@ def _start_server(self):
         """Start a sanic server."""
         debug = self.log_level == "DEBUG"
         app.run(
-            host="0.0.0.0", port=self.port, workers=self.n_workers, debug=debug, access_log=debug
+            host=self.host, port=self.port, workers=self.n_workers, debug=debug, access_log=debug
```

Comment:
Why?

Reply:
I saw there was a host option in the config that wasn't being used so I thought it might be an omission. If we don't want this to be configurable, I could instead make it a constant at the top of the file, so that the scheduler can pull it up to know where to send the requests.

Reply:
I added the host option for the same reason: to tell the client where the coco server is. According to Richard setting the host parameter here will limit from which interfaces requests are accepted. `0.0.0.0` means no restrictions and that's what we probably want here.

Reply:
OK, that makes sense. So I will restore `0.0.0.0` but keep the line that pulls the host from the config to initialise the scheduler. I'll leave a comment here because this had me confused.

Reply:
Or I could get rid of the host config parameter in `master` and just use `localhost` to initialise the scheduler.

Reply:
I think the scheduler will by design always run on the same host, so it can use `"localhost"`. Then only coco-client needs to read `host` from the config.

We might want to think about giving the client a separate config.